quizsolution.ipynb

```diff
@@ -0,0 +1,161 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "44aa06e6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import csv"
```

Comment:
write onther way to import csv file ?

Reply:
with open througt you can open csv file.....
it's a second methord....

Reply:
with open throug you can import csv file...
it's second methord to import file...